ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.13-nightly-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -30,8 +30,8 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- always_run: false
-  as: perfscale-aws-5nodes-loaded-upgrade-412to413-periodic
+- as: perfscale-aws-5nodes-loaded-upgrade-412to413-periodic
```

Comment:
should we just call this `5nodes-loaded-upgrade-412to413-periodic`, given the direction of some of the naming we have going on?